cbmc/proofs/IotMqtt_Connect/Makefile

```diff
@@ -0,0 +1,115 @@
+ENTRY=IotMqtt_Connect
+
+# We abstract these functions because manual inspection demonstrates they are unreachable
+ABSTRACTIONS += --remove-function-body _IotMqtt_RemoveSubscriptionByPacket
+ABSTRACTIONS += --remove-function-body _checkRetryLimit
+ABSTRACTIONS += --remove-function-body _scheduleCallback
+ABSTRACTIONS += --remove-function-body _scheduleNextRetry
+
+# We abstract all unreachable functions to improve coverage metrics
+ABSTRACTIONS += --remove-function-body IotNetworkInterfaceCloseCallback
+ABSTRACTIONS += --remove-function-body _mqttSubscription_setUnsubscribe
+ABSTRACTIONS += --remove-function-body _mqttOperation_match
+ABSTRACTIONS += --remove-function-body _matchEndWildcards
+ABSTRACTIONS += --remove-function-body _matchWildcards
+ABSTRACTIONS += --remove-function-body _packetMatch
+ABSTRACTIONS += --remove-function-body _topicFilterMatch
+ABSTRACTIONS += --remove-function-body IotTaskPool_ScheduleDeferred
+ABSTRACTIONS += --remove-function-body _IotMqtt_NextPacketIdentifier
+ABSTRACTIONS += --remove-function-body IotNetworkInterfaceReceive
+ABSTRACTIONS += --remove-function-body _IotMqtt_ScheduleOperation
+ABSTRACTIONS += --remove-function-body _IotMqtt_PublishSetDup
+
+OBJS += $(ENTRY)_harness.goto
+OBJS += $(MQTT)/cbmc/proofs/mqtt_state.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_api.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_helper.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_operation.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_serialize.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_subscription.goto
+OBJS += $(MQTT)/libraries/standard/mqtt/src/iot_mqtt_validate.goto
+
+# This should be smaller than CLIENT_IDENTIFIER_LENGTH_MAX for coverage
```

Comment:
This is a long list of assumptions.  Maybe the Makefile should have section markers before and after this with a comment at the top making clear that these are proof assumptions.